iceoryx_posh/include/iceoryx_posh/internal/mepoo/mepoo_segment.inl

```diff
@@ -72,7 +73,23 @@ inline SharedMemoryObjectType MePooSegment<SharedMemoryObjectType, MemoryManager
 {
     return std::move(
         typename SharedMemoryObjectType::Builder()
-            .name(writerGroup.getName())
+            .name([&writerGroup] {
+                using ShmName_t = detail::PosixSharedMemory::Name_t;
+                ShmName_t shmName = iceoryxResourcePrefix(roudi::DEFAULT_UNIQUE_ROUDI_ID,
+                                                          "p"); // use an additional 'p' to prevent creating a payload
```

Comment:
Why does a `p` prevent it? What else does `p` prevent or even allow ;).

Reply:
The management segment will be called `iox1_0_management`. Now lets assume there is a user called `management`. The payload segment would then also be called `iox1_0_management`. By adding a customizer, this is not possible anymore, since it would become `iox1_0_p_management`.

We also have this problem with the current code base and if someone uses `iceoryx_mgmt` as user name, that user would not be able to use iceoryx.